io.github.lxndr.gswatcher.yml

```diff
@@ -0,0 +1,47 @@
+app-id: io.github.lxndr.gswatcher
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: gswatcher
+finish-args:
+  - '--device=dri'
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--share=network'
+modules:
+  - shared-modules/lua5.4/lua-5.4.json
+  - name: geoip
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/maxmind/geoip-api-c/archive/v1.6.12.tar.gz
+        sha256: 99b119f8e21e94f1dfd6d49fbeed29a70df1544896e76cd456f25e397b07d476
+      - type: file
+        path: ./GeoIP.dat.gz
+        sha256: 14f3bcedc3c9da8369948a0e15a962122b7b93b02e2fc2bfa3eb75e7556ecc9e
+      - type: file
+        path: ./GeoIPCity.dat.gz
+        sha256: 7f1c52f6979b149165c7fbeccd7c7be51f17bea46f74403609c3cf616aed8148
+    post-install:
+      - gzip -d ./GeoIP.dat.gz
+      - gzip -d ./GeoIPCity.dat.gz
+      - install -d $FLATPAK_DEST/share/GeoIP
+      - install -m644 -t $FLATPAK_DEST/share/GeoIP GeoIP.dat GeoIPCity.dat
+  - name: gswatcher
```

Comment:
In appdata can you pull the screenshots from a tag instead of main https://github.com/lxndr/gswatcher/blob/3793a45ee36ae2fb3b54ce1e7ea5a8ac1933dae9/data/gswatcher.appdata.xml.in.in#L30 Thanks

Reply:
done